sources/Makefile

```diff
@@ -241,6 +241,12 @@ workers:
 	@docker compose -f docker-compose.yml build workers
 	@docker compose -f docker-compose.yml up -d workers
 
+# Runs db-migrations in REPL mode
+db-migrations-repl:
+	@echo "+\n++ Make: Database migrations REPL"
+	@docker compose -f docker-compose.yml build db-migrations-repl
+	@docker compose -f docker-compose.yml up -d db-migrations-repl
+
```

Comment:
What could be the benefit of using the DB Migration REPL in local?

Reply:
To reproduce the Openshift behavior and support the development 😉 
It was useful to check the behavior inside a docker container and reach a good level before testing on Openshift.